src/packs/ranged/Laser Pistol.yml

```diff
@@ -1,16 +1,13 @@
 _id: S69XOZtM17punHP7
 data:
-  ab: 0
+  ab: 1
   ammo:
     burst: false
     longReload: false
     max: 10
     suppress: false
     type: typeAPower
     value: 10
-  bundle:
-    amount: 1
-    bundled: false
```

Comment:
Don't update or remove data unrelated to PR

Reply:
This one was handled in #34.